### PR TITLE
Add Several components regarding Payment, customer details. 

### PR DESCRIPTION
**Pull request by erezshasha**

### CODE REVIEW

File: src/_codux/boards/checkout/checkout.board.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        <ComponentWrapper settings={{}}>
+            <ContentSlot>
+                <Checkout />
+            </ContentSlot>
+        </ComponentWrapper>
+    ),
+    isSnippet: true,
+    environmentProps: {
+        canvasMargin: { right: 0, bottom: 0, left: 0, top: 0 },
+        windowWidth: 382,
```
Comment (YuriyMorkovnik) [inline src/_codux/boards/checkout/checkout.board.tsx:17]: can remove this

File: src/_codux/boards/payment/payment.board.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            <ContentSlot>
+                <Payment />
+            </ContentSlot>
+        </ComponentWrapper>
+    ),
+    isSnippet: true,
+    environmentProps: {
+        canvasMargin: { right: 0, bottom: 0, left: 0, top: 0 },
+        windowWidth: 366,
+        windowHeight: 1060,
```
Comment (YuriyMorkovnik) [inline src/_codux/boards/payment/payment.board.tsx:18]: can remove this

File: src/_codux/boards/single-input/single-input-mandatory.board.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        <ComponentWrapper settings={{}}>
+            <ContentSlot>
+                <SingleInput inputTitle="First name" hasError={true} isMandatory={true} />
+            </ContentSlot>
+        </ComponentWrapper>
+    ),
+    isSnippet: true,
+    environmentProps: {
+        canvasMargin: { right: 0, bottom: 0, left: 0, top: 0 },
+        windowWidth: 914,
```
Comment (YuriyMorkovnik) [inline src/_codux/boards/single-input/single-input-mandatory.board.tsx:17]: can remove this

File: src/components/checkout/checkout.module.scss [hunk 4]
```diff
@@ -0,0 +1,18 @@
+@import '@styles/theme.module.scss';
+
+.root {
+    margin: 10px;
+}
+.button {
+    background-color: #131212;
```
Comment (YuriyMorkovnik) [inline src/components/checkout/checkout.module.scss:7]: let's use css variables

File: src/components/checkout/checkout.module.scss [hunk 5]
```diff
@@ -0,0 +1,18 @@
+@import '@styles/theme.module.scss';
+
+.root {
+    margin: 10px;
+}
+.button {
```
Comment (YuriyMorkovnik) [inline src/components/checkout/checkout.module.scss:6]: do we really need a new button? why not use a generic one from common-styles.module.scss?

File: src/components/customer-details-preview/customer-details-preview.module.scss [hunk 6]
```diff
@@ -0,0 +1,18 @@
+@import '@styles/theme.module.scss';
+
+.root {
+}
+.customerDetailsPreview {
+    display: flex;
+    flex-direction: column;
+    color: $charcoal-black;
+}
+.dddd {
```
Comment (YuriyMorkovnik) [inline src/components/customer-details-preview/customer-details-preview.module.scss:10]: it's not used

File: src/components/customer-details-preview/customer-details-preview.module.scss [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    display: flex;
+    flex-direction: column;
+}
+.CustomerDetailsTitle {
```
Comment (YuriyMorkovnik) [inline src/components/customer-details-preview/customer-details-preview.module.scss:14]: please don't capitalize class names

File: src/components/payment/payment.module.scss [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+    font-weight: bolder;
+    font-size: 18px;
+    margin-bottom: 10px;
+}
+.subTitle {
+    margin-top: 30px;
+    margin-bottom: 10px;
+    font-weight: 500;
+}
+.btnPayment {
```
Comment (YuriyMorkovnik) [inline src/components/payment/payment.module.scss:20]: again, do we really need a new button? maybe to use existing one and combine missing styles with classnames lib?

File: src/components/payment/payment.module.scss [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+    width: 100%;
+    text-align: center;
+    background-color: #5783d7;
```
Comment (YuriyMorkovnik) [inline src/components/payment/payment.module.scss:23]: var?

File: src/components/payment/payment.module.scss [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+    background-position: 0%;
+    background-size: auto;
+    background-repeat: repeat;
+    background-attachment: scroll;
+    background-origin: padding-box;
+    background-clip: border-box;
+    margin-top: 10px;
+    margin-bottom: 10px;
+    height: 50px;
+    color: #f0f2ec;
```
Comment (YuriyMorkovnik) [inline src/components/payment/payment.module.scss:34]: var?

File: src/components/payment/payment.tsx [hunk 11]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+export const Payment = ({ className }: PaymentProps) => {
+    return (
+        <div className={classNames(styles.root, className, styles.paymentdiv)}>
+            <h3 className={styles.title}>Payment</h3>
+            <h2 className={styles.boldMessage}>
+                We&apos;re not able to accept online payments at the moment.
+            </h2>
+            <h3>Please contact us to complete your purchase. Thanks. </h3>
```
Comment (YuriyMorkovnik) [inline src/components/payment/payment.tsx:17]: you use heading elements (<h /> )everywhere. I don't think we really need it. Replace some of them with <span/> or whatever is semantically more appropriate.

File: src/components/single-input/single-input.tsx [hunk 12]
```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames';
+import styles from './single-input.module.scss';
+import commonStyles from '@styles/common-styles.module.scss';
+
+export interface SingleInputProps {
+    className?: string;
+    inputTitle: string;
```
Comment (YuriyMorkovnik) [inline src/components/single-input/single-input.tsx:7]: label looks more appropriate

File: src/components/single-input/single-input.tsx [hunk 13]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+export const SingleInput = ({ className, inputTitle, hasError, isMandatory }: SingleInputProps) => {
+    const styleToApply = hasError ? styles.inputError : styles.input;
+    const styleToApply1 = styles.input;
+
+    return (
+        <div className={classNames(styles.root, className)}>
+            {inputTitle}
+            {isMandatory ? ' *' : true}
```
Comment (YuriyMorkovnik) [inline src/components/single-input/single-input.tsx:19]: {required ? ' *' : null}

File: src/components/single-input/single-input.module.scss [hunk 14]
```diff
@@ -0,0 +1,20 @@
...
+    display: flex;
+    flex-direction: column;
+    margin: 8px 0px;
+}
+.input {
+    border: 1px solid #b3b3b3;
+    padding-left: 8px;
+    height: 30px;
+}
+.errormessage {
```
Comment (YuriyMorkovnik) [inline src/components/single-input/single-input.module.scss:13]: camelCase

File: src/_codux/boards/checkout/checkout.board.tsx [hunk 15]
```diff
@@ -0,0 +1,19 @@
+import { Checkout } from '../../../components/checkout/checkout';
+import { ContentSlot, createBoard } from '@wixc3/react-board';
+import { ComponentWrapper } from '/src/_codux/board-wrappers/component-wrapper';
+
+export default createBoard({
+    name: 'Checkout',
+    Board: () => (
+        <ComponentWrapper settings={{}}>
```
Comment (YuriyMorkovnik) [inline src/_codux/boards/checkout/checkout.board.tsx:8]: we don'r really need to pass an empty object

File: src/components/single-input/single-input.tsx [hunk 16]
```diff
@@ -0,0 +1,26 @@
...
+
+export interface SingleInputProps {
+    className?: string;
+    inputTitle: string;
+    hasError?: boolean;
+    isMandatory?: boolean;
+}
+
+export const SingleInput = ({ className, inputTitle, hasError, isMandatory }: SingleInputProps) => {
+    const styleToApply = hasError ? styles.inputError : styles.input;
```
Comment (YuriyMorkovnik) [inline src/components/single-input/single-input.tsx:13]: I'd prefer to dynamically add additional styles this way
```
className={classNames(styles.input, {
                            [styles.inputError]: hasError,
                        })}
```

**Comment by YuriyMorkovnik** [comment]
add more detailed description please

File: src/_codux/boards/checkout/checkout.board.tsx (hunk 1 above)
Comment (erezshasha) [reply src/_codux/boards/checkout/checkout.board.tsx:17]: there are the board's props - why it should be removed?

**Review by YuriyMorkovnik — APPROVED** [review]

**Pull request merged**